libopenage/unit/attribute.h

```diff
@@ -115,7 +115,11 @@ class AttributeContainer {
 
 using attr_map_t = std::map<attr_type, std::shared_ptr<AttributeContainer>>;
 
-using typeamount_set = std::map<int, unsigned int>;
+/**
+ * An unordered_map with a int key used as a type id
+ * and a unsigned int value used as the amount
+ */
+using typeamount_map = std::unordered_map<int, unsigned int>;
```

Comment:
Yes, but i didn't write that on the doc because it can be used for other things too.